job_templates/ansible_roles_-_install_from_git.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+name: Ansible Roles - Install from git
+job_category: Ansible Roles Installation
+description_format: Clone roles from git repository to %{location}
+snippet: false
+template_inputs:
+- name: git_repository
+  required: true
+  input_type: user
+  description: "URL to the git repository containing the roles, e.g:\r\nhttps://github.com/theforeman/foreman_role_1"
+  advanced: false
+- name: location
+  required: true
+  input_type: user
+  description: For example, '/etc/ansible/roles/foobar' . Look at '/etc/ansible/ansible.cfg'
+    roles_path option to find what is your roles_path
+  advanced: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - command: git clone <%= input('git_repository') %> <%= input('location') %>
```

Comment:
just curious, not necessary to fix in this PR, is there a reason why we don't use [native git module ](http://docs.ansible.com/ansible/latest/git_module.html) and fallback to shell command?

Reply:
Actually no reason but ignorance (I just discovered it), I'm not sure how that'd work for multiple items which is what I had in mind originally